docs/cli/function/build-options.md

```diff
@@ -3,6 +3,16 @@ title: Build options
 description: Use build options for Amplify's function category to execute a script before a function is deployed, e.g. to transpile Typescript or ES6 with Babel into a format that is supported by the AWS Lambda's node runtime.
 ---
 
+## Python
+
+**There are no existing build options for Python functions**
```

Comment:
Remove the bold lettering here and move it and collapse it into the next paragraph as the first sentence.